CircularProgressBar/script.js

```diff
@@ -1,8 +1,9 @@
 const slider = document.getElementById("slider");
 const border = document.getElementById("circleBorder");
 const progress = document.getElementById("showProgress");
-slider.addEventListener("input",()=>{
-    let degrees = slider.value * 3.6;
-    border.style.background = "conic-gradient(rgb(6, 214, 160) "+degrees+"deg, black 0deg)";
-    progress.innerHTML = slider.value+"%";
+
+slider.addEventListener("input", () => {
+    const percentage = slider.value;
+    border.style.background = "conic-gradient(rgb(6, 214, 160) " + percentage + "deg, black 0deg)";
+    progress.innerHTML = slider.value + "%";
```

Comment:
Isto vai dar errado, e esta um pouco redundante 
```suggestion
    border.style.background = "conic-gradient(rgb(6, 214, 160) " + slider.value + "%, black 0%)";
    progress.innerHTML = slider.value + "%";
```

Reply:
Tinhas esquecido de trocar de `deg` para `%`.
E estavas a criar uma variavel praticamente para nada.